ifopt_core/src/leaves.cc

```diff
@@ -50,20 +50,26 @@ ConstraintSet::GetJacobian () const
   Jacobian jacobian(GetRows(), variables_->GetRows());
 
   int col = 0;
+  Jacobian jac = Jacobian(GetRows(), 1);
```

Comment:
This should probably just be `Jacobian jac;` to not duplicate the initialization done in line 58.